### ISSUE TITLE
Suggestion to add Websocket support

### ISSUE THREAD

**Issue by LucasBoberg** [L2jLiga/fastify-decorators#9]
Hi!

It would be nice to add websocket support using fastify-websocket.

Currently most of it seams to work except getting params.

**Comment by L2jLiga** [comment]
Hi!

Can you explain little bit better what is wrong or how you would like to see it?

**Comment by LucasBoberg** [comment]
Hi!

I would like to be able to do something like this:

```
 @GET({ url: "/shop/realtime/:id", options: { websocket: true, schema: {
    tags: ["order"]
  }}})
  async getRealtimeOrdersByShop(connection, request, params) {
    try {
      
      const id = await params.id;
      console.log(connection);
      const orderRepository = await getManager().getCustomRepository(OrderRepository);
      const orders = await orderRepository.findByShopId(id);
      connection.socket.send(JSON.stringify(orders))
      
    } catch (error) {
      throw boom.boomify(error);
    }
  }
```

Currently params returns undefined but using the regular way to register a route as stated in the docs, params works perfectly

**Comment by L2jLiga** [comment]
released as v2.0.0-1 under next tag, thanks!

**Comment by LucasBoberg** [comment]
Thanks for the super quick and awsome work!
